src/components/keyboard-aware-flat-list.ios.js

```diff
@@ -0,0 +1,65 @@
+import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view';
+/**
+* @format
```

Comment:
Small nitpicking:
Can we have the `import` bellow those comments?

Reply:
I put this here as a solution to flow error "Cannot resolve module `react-native-keyboard-aware-scroll-view`"
I think that happens because I added `react-native-keyboard-aware-scroll-view` to flowconfig ignore list because flow was trying to validate library source code as well. if there's another solution to this I can apply that happily I just couldn't find any.

Reply:
I see, that makes sense. In that case can we put a small comment explaining that? :)

Reply:
@hypest offered a new solution and I've pushed a new commit for it https://github.com/wordpress-mobile/gutenberg-mobile/pull/387/commits/42540721435b273b7d5ee6afcea582325f59ffe6 It is so much better this way 🎉

Reply:
Great solution! 😁